tenable/io/v3/vm/networks.py

```diff
@@ -0,0 +1,329 @@
+                The resource record of the newly created network.
+
+        Examples:
+            >>> nw = tio.v3.vm.networks.create('Example')
+        '''
+        if not description:
+            description = ''
+
+        return self._post('', json={
+            'name': name,
+            'description': description,
+            'assets_ttl_days': assets_ttl_days
+        })
+
+    def delete(self, network_id: str) -> None:
+        '''
+        Deletes the specified network.
+
+        :devportal:`networks: delete <networks-delete>`
+
+        Args:
+            network_id: The UUID of the network to remove.
+
+        Examples:
+            >>> tio.v3.vm.networks.delete(
+                    '00000000-0000-0000-0000-000000000000'
+                )
+        '''
+        self._delete(f'{network_id}')
+
+    def details(self, network_id: str) -> Dict:
+        '''
+        Retrieves the details of the specified network.
+
+        :devportal:`networks: details <networks-details>`
+
+        Args:
+            network_id : The UUID of the network.
+
+        Examples:
+            >>> nw = tio.v3.vm.networks.details(
+                    '00000000-0000-0000-0000-000000000000'
+                )
+        '''
+        return self._get(f'{network_id}')
+
+    def edit(self,
+             network_id: str,
+             name: str,
+             description=None,
+             assets_ttl_days=None
+             ) -> Dict:
+        '''
+        Updates the specified network resource.
+
+        :devportal:`networks: update <networks-update>`
+
+        Args:
+            network_id: The UUID of the network resource to update.
+            name: The new name of the network resource.
+            description (str, optional):
+                The new description of the network resource.
+            assets_ttl_days (int, optional):
+                The number of days to wait before assets age out.
+                Assets will be permanently deleted if they are not seen on a
+                scan within the specified number of days.
+                Minimum value: 90
+                Maximum value: 365
+
+        Returns:
+            :obj:`dict`:
+                The updates network resource.
+
+        Examples:
+            >>> nw = tio.v3.vm.networks.edit(
+                    '00000000-0000-0000-0000-000000000000',
+                    'Updated Network', 'Updated Description', 180
+                )
+        '''
+        if not description:
+            description = ''
+
+        return self._put(f'{network_id}',
+                         json={
+                             'name': name,
+                             'description': description,
+                             'assets_ttl_days': assets_ttl_days
+                         }
+                         )
+
+    def assign_scanners(self,
+                        network_id: str,
+                        *scanner_uuids: Union[str, List]
+                        ) -> None:
```

Comment:
You can use NoReturn type from typing module here

Reply:
updated as per our latest discussion.